physionet-django/project/models.py

```diff
@@ -858,18 +824,104 @@ def citation_text(self):
 
             elif style == 'Vancouver':
 
+                style_format = ('{author}. {title} (version {version}). '
+                                'PhysioNet. {year}')
+
+                doi_format = ('. Available from: '
+                              '<a href="https://doi.org/{doi}">'
+                              'https://doi.org/{doi}</a>.')
+
                 all_authors = ', '.join(a.initialed_name()
-                                         .replace(',', '')
-                                         .replace('.', '') for a in authors)
+                                        .replace(',', '')
```

Comment:
Something that Benjamin suggested was to update the initialed_name function, so that it takes arguments (instead of fixing the string it returns).  I think this would be a good idea!

Reply:
I will do it, I think this is the only instance it is needed but I suppose it may be useful further down for something else.